ldap-authenticator/src/main/java/org/xwiki/contrib/ldap/XWikiLDAPUtils.java

```diff
@@ -833,9 +833,9 @@ public Map<String, String> getGroupMembers(String groupDN, XWikiContext context)
                     LOGGER.debug("Retrieving Members of the group [{}]", groupDN);
 
                     boolean isGroup = getGroupMembers(groupDN, members, new ArrayList<String>(), context);
+                    groupMembers = members;
```

Comment:
I'm not a fan of this change. getGroupMembers is supposed to return null if no group matches the passed name. If you hit a use case where a this is a group and getGroupMembers returns false then that's what needs to be fixed.